generic/tests/cfg/unattended_install.cfg

```diff
@@ -43,6 +43,11 @@
     # This value can be set to 'yes' if required to perform installation trigger
     # random time in the range random.randint(0, install_trigger_time)
     random_trigger = "no"
+    ovmf:
```

Comment:
Already included in VT.
https://github.com/avocado-framework/avocado-vt/blob/6681e9245af2f2c130775c1a0d7eab134216c553/virttest/shared/cfg/guest-hw.cfg#L460-L470

Reply:
remove same.